src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -30,7 +30,7 @@ Starting July 30, 2020, all of our new customers are on a pricing plan that we c
 For New Relic One pricing, billing is based on these factors:
 
 * The pricing tier (Standard, Pro, Enterprise). Higher pricing tiers give access to more account-related admin features, more support, longer data retention, and other features ([learn more](https://newrelic.com/pricing)).
-* The number of [full users](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) (users with access to everything available at that tier). [Basic users](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) are free. Standard tier includes one full user for free, and a max of five.
+* The count of provisioned [full users](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) (users with access to everything available at that tier). [Basic users](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) are free. Standard tier includes one full user for free, and a max of five.
 * The amount of data ingested. 100 GBs per month is free. $0.25 per GB ingested above that.
```

Comment:
I'm tempted to want to move the data ingested bullet above the user one. It's so much easier to understand :)